sdk/storage/storage-blob/src/IBlobSASSignatureValues.ts

```diff
@@ -161,22 +220,138 @@ export interface IBlobSASSignatureValues {
 export function generateBlobSASQueryParameters(
   blobSASSignatureValues: IBlobSASSignatureValues,
   sharedKeyCredential: SharedKeyCredential
+): SASQueryParameters;
+/**
```

Comment:
Minor: Could we do overload with nicer format?

Reply:
Will add several empty lines between them.